e2e/tests/steps_init_test.go

```diff
@@ -45,6 +52,27 @@ type rootSuite struct {
 	// validator holds command results between runs and reports errors to the test suite
 	validator *validatorPod
 	// validatorA maps to suffix ID 001 of the kube pod that we use as our control agent
+
+	// servicerKeys is hydrated by the clientset with credentials for all servicers.
+	// servicerKeys maps servicer IDs to their private key as a hex string.
+	servicerKeys map[string]string
+
+	// appKeys is hydrated by the clientset with credentials for all apps.
+	// appKeys maps app IDs to their private key as a hex string.
+	appKeys map[string]string
+
+	// relaychains holds settings for all relaychains used in the tests
+	//	the map key is a constant selected as the identifier for the relaychain, e.g. "RelayChainETH" for Ethereum
+	relaychains map[string]*relaychainSettings
+
+	// servicer holds the key for the servicer that should received the relay
+	servicer string
```

Comment:
Is this a key or an address? Please add the appropriate suffix so I think it should be `ethServicerKey`

Reply:
Thank you for the review. I updated the name to `servicerKey` as this could hold the key for a servicer matching any service/relay chain. Please let me know if the `ethServicerKey` seems a better option.